internal/executor/instance/instance.go

```diff
@@ -122,12 +122,16 @@ func RunDockerizedAgent(ctx context.Context, config *RunConfig, params *Params)
 	}
 
 	if config.DockerOptions.ShouldPullImage(params.Image) {
-		logger.Debugf("pulling image %s", params.Image)
+		dockerPullLogger := logger.Scoped("docker pull")
+		dockerPullLogger.Infof("pulling image %s", params.Image)
 		progress, err := cli.ImagePull(ctx, params.Image, types.ImagePullOptions{})
 		if err != nil {
+			dockerPullLogger.Errorf("Failed to pull %s: $v", params.Image, err)
```

Comment:
```suggestion
			dockerPullLogger.Errorf("failed to pull %s: %v", params.Image, err)
```

Reply:
Hold on, this is a user visible error level log